src/verifier.js

```diff
@@ -78,7 +78,7 @@ class LocalVerifier {
         const payload = { [`${this.options.entity}Id`]: id };
         done(null, entity, payload);
       })
-      .catch(error => error ? done(error) : done(null, error));
+      .catch(error => error ? done(error) : done(null, error, { message: 'Invalid login' }));
```

Comment:
Ya so I think we can go either way here. For local auth it really doesn't matter. The `fail` method is used more with OAuth and OpenID flows because there are more ways that the authentication can actually fail that may not be a result of bad user credentials. https://github.com/jaredhanson/passport-oauth2/blob/master/lib/strategy.js#L129